index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sora|Tangerine|Inconsolata|Droid+Sans">
+    <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css" class="src">
+    <script src="index.js"></script>
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="main">
+      <div class="top-navigation">
+        <div class="wrapper">
+          <ul>
+            <li class="name"><span>pius ssekweyama</span><i class="fa fa-bars" aria-hidden="true"></i></li>
+            <li>portfolio</li>
+            <li>about</li>
+            <li>contact</li>
+            <li class="social">
+              <i class="fa fa-envelope" aria-hidden="true"></i>
+            </li>
+          </ul>
+        </div>
```

Comment:
- Please the container for this section should be a `header` tag, and your navigation (logo and menu) should be wrapped in a `nav` tag. This would help improve your site's HTM semantics and improve accessibility 👍 👍

Reply:
Good job following the advice of the previous reviewer